components/next_steps_view/steps/invite_members_step/invite_members_step.tsx

```diff
@@ -224,7 +258,10 @@ export default class InviteMembersStep extends React.PureComponent<Props, State>
                             </h3>
                             <FormattedMessage
                                 id='next_steps_view.invite_members_step.youCanInviteUpTo'
-                                defaultMessage='You can invite up to 10 team members using a space or comma between addresses'
+                                defaultMessage='You can invite up to {members} team members using a space or comma between addresses'
```

Comment:
Here I believe we actually just want to show what is the max they can have in their free tier and not how much they have remaining. Posted a question in the channel to get some input on what is the best message here: https://community-daily.mattermost.com/private-core/pl/ao8ghbr4oiyiimy8eor1eqrtky

Reply:
Based on Adam's respond on that thread, this is showing the correct remaining users. Will resolve this comment.